src/app.module.ts

```diff
@@ -36,11 +40,18 @@ import { UsersService } from './users/users.service'
         ),
     }),
     GraphQLModule.forRootAsync({
-      imports: [ConfigModule, UsersModule],
-      inject: [ConfigService, UsersService],
+      imports: [ConfigModule, UsersModule, AuthenticationModule],
```

Comment:
can someone confirm that this looks right?

Reply:
Yea looks good - since you use `AuthenticationService` in `graphqlConfigFactory` to get the user from cookie, you need to inject it here as you've done.